### DISCUSSION TITLE
[Feature] Support CloudMediaProvider

### DISCUSSION THREAD

**Discussion by RealJohnGalt** [immich-app/immich#10757] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Support https://developer.android.com/reference/android/provider/CloudMediaProvider in the android mobile app to allow picking from immich in MediaProvider media picker. As a note, the allowlist check android platform side must be relaxed or it must be added to allowlist.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#7101

**Reply by RealJohnGalt** [reply]
> #7101

Thank you, I didn't find it when I searched somehow. I will close this.
